url/interfaces.any.js

```diff
@@ -28,7 +21,7 @@
   USVString toJSON();
 };
 
-[Constructor(optional (USVString or URLSearchParams) init = ""),
+[Constructor(optional (sequence<sequence<USVString>> or USVString) init = ""), // or record<USVString, USVString>
```

Comment:
Note that including `record<>` broke IDL harness. Who would know if that's a known issue? @heycam?